### ISSUE TITLE
Timeline: A11y - no text alternative available for timeline item icon

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents#12238]
### Describe the bug

The icon information is not conveyed programmatically to screen reader users.

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

The information which is conveyed via shape should have an alternative.

### UI5 Web Components for React Version

2.13.2

### UI5 Web Components Version

2.13.2

### Browser

Chrome

### Operating System

MacOS

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by ilhan007** [comment]
Hello @summer2200 can you provide some example, or enhance [this one](https://tinyurl.com/2k8z2cb5) to better understand the problem.

**Comment by summer2200** [comment]
Hi @ilhan007 , this was actually raised by our customer, I will update here when I get their reply.

**Comment by summer2200** [comment]
Hi @ilhan007 ,

The customer thought "aria-label" should be added to the icon, so they can get some more information of different timeline items. Is there any possible to support adding aria-label to the icon?

For the steps to reproduce, please refer to the slack channel.

Thanks,
Summer

**Comment by ilhan007** [comment]
The Icon itself has accessibleName prop
https://ui5.github.io/webcomponents/components/Icon/#accessiblename

The thing is that the TimelineItem does not provide a slot to aggregate an icon, image or anything from outside that you can set attributes like aria-label on it, neither iconAccessibleName prop that could be propagated to the ui5-icon that is internally used.

So it's a feature request to the TimelineItem.
Hello @UI5/ui5-team-ctr-bal could you evaluate if api enhancement should be done, if it's required, given the case provided by @summer2200 (also application details are provided in the slack channel to explore the case) and process further.

**Comment by Joycelin02** [comment]
Hi @tsanislavgatev @ilhan007, 

Please see the [Fiori guideline](https://www.sap.com/design-system/fiori-design-web/ui-elements/timeline/?external#post-entryfeed-update) and [Fiori Spec](https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2698915855) for Timeline.

[A11y Guideline for icon](https://sap.sharepoint.com/sites/205197/SitePages/Avoid-Verbose-for-Screen-Readers.aspx)
**Invisible Label:**Used as an alternative for the icons. Repeat the aria-label (invisible label) in a tooltip to provide the same information to sighted users.

A11y Guideline_tooltip for icon
https://www.figma.com/design/8NlRC1AgadiPLcb0OSProX/Accessibility-Design-Tools-V1-%E2%80%93-Internal?m=auto&node-id=3810-26132&t=8YNan4skKspJvpPU-1

Please let me know if you need further info. 

Thanks,
Joyce

**Comment by DMihaylova** [comment]
Hi @summer2200,

Whether accessibleName or accessibleDescription on a ui5-timeline-item would suit your needs?
If yes, would you also need accessibleNameRef and accessibleDescriptionRef?

@Joycelin02, the icon here is decorative, therefore, as far as I know, no need to be announced or shown in the tooltip. On top some of the icons don't have names.
If there's an accessible name or description on a timeline item level, the application could add the needed text (which could be different from the icon's technical name) so it's announced. Correct me if I miss something :)

Regards,
Diana

**Comment by Joycelin02** [comment]
Hi @DMihaylova,

The icons here are not just decorative; they carry meaning. We received a ticket from a customer who doesn't understand the meaning of the icons. Could we add an aria-label to the icons for accessibility purposes? That would be helpful for users. Also, Please refer to the guidelines for icons to ensure it's not a violation of accessibility (a11y) standards. Thank you!

Tooltip & a11y for step status icons in route map (timeline):
<img width="749" height="866" alt="Image" src="https://github.com/user-attachments/assets/5f4ccfd5-fdca-499d-9122-934fd03b2227" />


[A11y Guideline for icon](https://sap.sharepoint.com/sites/205197/SitePages/Avoid-Verbose-for-Screen-Readers.aspx)
<img width="993" height="751" alt="Image" src="https://github.com/user-attachments/assets/51bb263d-866f-4aa0-bf60-4a3457eda291" />

A11y Guideline_tooltip for icon
https://www.figma.com/design/8NlRC1AgadiPLcb0OSProX/Accessibility-Design-Tools-V1-%E2%80%93-Internal?m=auto&node-id=3810-26132&t=8YNan4skKspJvpPU-1
<img width="1415" height="832" alt="Image" src="https://github.com/user-attachments/assets/74688211-6024-4df2-9141-4c15e624a1eb" />

Thanks,
Joyce

**Comment by Joycelin02** [comment]
Hi @DMihaylova,
I also raised this issue to a11y expert, and here is her feedback:
<img width="985" height="515" alt="Image" src="https://github.com/user-attachments/assets/52e0021a-1cbc-44b8-84a4-08e6a222e1e1" />
Please add comment [here](https://jira.tools.sap/browse/PMU-25957) or contact her if you have any concern. 
Email: irla.bocianoski.rebelo@sap.com

Thank you
Joyce

**Comment by DMihaylova** [comment]
Tracked internally as SAPDESIGNSYSTEM-462.

**Comment by judith9209** [comment]
Spec update is completed see comment in the inner source ticket. We discussed in yesterdays meeting to aim 2.22.0.
